packages/opentelemetry-metrics/src/Meter.ts

```diff
@@ -310,8 +312,23 @@ export class Meter implements api.Meter {
     return this._batcher;
   }
 
-  async shutdown(): Promise<void> {
-    await this._controller.shutdown();
+  shutdown(): Promise<void> {
```

Comment:
Why did you remove async?

Reply:
because it is returning `_shuttingDownPromise` this way the function can be called many times and will resolve once the shutdown is complete